docs/DeveloperGuide.md

```diff
@@ -1,37 +1,1036 @@
+An example of a success message would be
+
+```
+----------------------------------------------------------------------
+Schedule had been cleared and reset.
+There is no plan scheduled for any day.
+To add plan for any day, enter:
+schedule /update <day number [1-7]> <plan number>
+----------------------------------------------------------------------
+```
+
+(Step 10) `FileManager#rewriteAllDaysScheduleToFile(dayList)` is called to write all the Day objects' data stored in the dayList
+into `schedule.txt` which is stored on the user's local filesystem. Since all Day objects are deleted, the writing of
+data into `schedule.txt` would be an equivalent of resetting the text file. 
+
+This completes the process of clearing of all plans stored in the schedule on WerkIt!
+
+---
+### Search
+
+#### Search For Exercise
+Format: `search /exercise <keywords>`
+
+The `Parser#createSearchCommand(String userInput)` method will further evaluate the user input
+`/exercise` and call the constructor of `SearchCommand` class by passing relevant parameters related to search exercise
+to the constructor. The created `SearchCommand` object is returned by the `Parser#createSearchCommand(String userInput)`
+method to `Parser#parseUserInput(String userInput)` method, and finally returned by
+`Parser#parseUserInput(String userInput)` method to `WerkIt#startContinuousUserPrompt()` method. The search command will
+be executed in `WerkIt#startContinuousUserPrompt()`. And based on the `<keywords>` specified by the user, the output
+will either be a list of matching exercises or 'Sorry, no matching exercise found' if the user has entered the command
+correctly.
+
+The following sequence diagram illustrates how the `search /exercise` command works in greater detail:
+
+![Search Exercise Sequence Diagram](uml/sequenceDiagrams/images/searchExercise.png)
+
+#### Search For Workout
+Format: `search /workout <keywords>`
+
+The `Parser#createSearchCommand(String userInput)` method will further evaluate the user input
+`/workout` and call the constructor of `SearchCommand` class by passing relevant parameters to the constructor.
+The created `SearchCommand` object is returned by the `Parser#createSearchCommand(String userInput)`
+method to `Parser#parseUserInput(String userInput)` method, and finally returned by
+`Parser#parseUserInput(String userInput)` method to `WerkIt#startContinuousUserPrompt()` method. The search command will
+be executed in `WerkIt#startContinuousUserPrompt()`. And based on the `<keywords>` specified by the user, the output
+will either be a list of matching names of workout or 'Sorry, no matching workout found' if the user has entered the command
+correctly.
+
+The following sequence diagram illustrates how the `search /workout` command works in greater detail:
+
+![Search Exercise Sequence Diagram](uml/sequenceDiagrams/images/searchWorkout.png)
```

Comment:
The search /exercise, search /workout and search /plan commands seem to be very similar. Perhaps your team would like to remove all but one of them? It seems like if the reader has gotten this far in the developer guide, the calls to the different lists will be quite obvious.